src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-share-list/smb-share-list.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
+import { Observable, BehaviorSubject, of } from 'rxjs';
+import { TableComponent } from '~/app/shared/datatable/table/table.component';
+import { CdTableColumn } from '~/app/shared/models/cd-table-column';
+import { CdTableFetchDataContext } from '~/app/shared/models/cd-table-fetch-data-context';
+import { Permission } from '~/app/shared/models/permissions';
+import { SMBShare } from '../smb.model';
+import { switchMap, catchError } from 'rxjs/operators';
+import { SmbService } from '~/app/shared/api/smb.service';
+import { AuthStorageService } from '~/app/shared/services/auth-storage.service';
+import { CellTemplate } from '~/app/shared/enum/cell-template.enum';
+
+@Component({
+  selector: 'cd-smb-share-list',
+  templateUrl: './smb-share-list.component.html',
+  styleUrls: ['./smb-share-list.component.scss']
+})
+export class SmbShareListComponent implements OnInit {
+  @Input()
+  clusterId: string;
+  @ViewChild('table', { static: true })
+  table: TableComponent;
+  columns: CdTableColumn[];
+  permission: Permission;
+  context: CdTableFetchDataContext;
+
+  smbShares$: Observable<SMBShare[]>;
+  subject$ = new BehaviorSubject<SMBShare[]>([]);
+
+  constructor(private authStorageService: AuthStorageService, private smbService: SmbService) {
+    this.permission = this.authStorageService.getPermissions().smb;
+  }
+
+  ngOnInit() {
+    this.columns = [
+      {
+        name: $localize`ID`,
+        prop: 'share_id',
+        flexGrow: 2
+      },
+      {
+        name: $localize`Name`,
+        prop: 'name',
+        flexGrow: 2
+      },
+      {
+        name: $localize`File System`,
```

Comment:
```suggestion
        name: $localize`FileSystem`,
```

Reply:
I think it should be this, PTAL cc @bhkaur25

Reply:
Regardless, I went with `File System` as it is how it is displayed in the left-side menu as well as the `File System` page:
![image](https://github.com/user-attachments/assets/c76ad7c5-fcfe-4a3c-9442-0f18829dbc53)
![image](https://github.com/user-attachments/assets/20458e10-b0ad-4fc4-9985-5d36b1a84666)

This is consistent through the application